source/components/recipe-details/detailpage.html

```diff
@@ -0,0 +1,138 @@
+          <!-- flexbox includes info and category box-->
+          <div class="info-category-box">
+            <!-- infomation box-->
+            <div class="info-box">
+              <div class="info-title">Info:</div>
+              <div class="info-list">
+                <ul>
+                  <li>Cost: $5 per serving</li>
+                  <li>Time: 40min</li>
+                  <li>Serving Size: 5</li>
+                </ul>
+              </div>
+            </div>
+            <!-- category box-->
+            <div class="category-box">
+              <div class="category-title">Categories:</div>
+              <div class="category-list">
+                <li>Vegan</li>
+                <li id="low-calories">Low Calories</li>
+              </div>
+            </div>
+          </div>
+          <!-- nutrient box-->
+          <div class="nutrient-box">
+            <div class="nutrient-title">Nutrients (per serving):</div>
+            <div class="nutrient-list">
+              <ul>
+                <li>250 Calories</li>
+                <li>4 grams of protein</li>
+                <li>3 grams of sugar</li>
+                <li>3 miligrams of salt</li>
+                <li>5 grams of protein</li>
+              </ul>
+            </div>
+          </div>
+        </div>
+        <!-- ingredients box-->
+        <div class="ingredients-box">
+          <div class="ingredients-title">Ingredients:</div>
+          <div class="ingredients-list">
+            <ul>
+              <li>3 cups of all purpose blour</li>
+              <li>whatsup</li>
+              <li>ketchup</li>
+              <li>water</li>
+              <li>water</li>
+              <li>food</li>
+              <li>food</li>
+              <li>potato</li>
+              <li>tomato</li>
+            </ul>
+          </div>
+        </div>
+
+        <!-- Direction box-->
+        <div class="direction-box">
+          <div class="direction-title">Directions:</div>
+          <div class="direction-list">
+            <ol>
+              <li>3 cups of all purpose flour</li>
+              <li>
+                Heat Milk-Butter: While egg is beating, place butter and mile
+                ina heatproof jug
+              </li>
+              <li>
+                Whisk flour, baking powder and salt in a large bowl. Set aside
+              </li>
+              <li>food</li>
+              <li>food</li>
+            </ol>
+          </div>
+        </div>
+        <!-- comment box-->
+        <div class="comment-box">
+          <div class="comment-box-input">
+            <div class="user-input">User input</div>
+            <div class="picture-box">pic</div>
+            <div class="submit-box">submit</div>
+          </div>
+
+          <div class="comment-box-reply">
+            <div class="comment-box-reply1">display user input here</div>
+            <div class="comment-box-reply2">display user input here</div>
+          </div>
```

Comment:
Maybe replace `comment-box-reply` with `comment-box-reply-container` and the `-reply1` and `-reply2` with just `-reply`? See below:
```suggestion
          <div class="comment-box-reply-container">
            <div class="comment-box-reply">display user input here</div>
            <div class="comment-box-reply">display user input here</div>
          </div>
```